doc/api/crypto.md

```diff
@@ -633,16 +938,44 @@ exchanges.
 Instances of the `DiffieHellman` class can be created using the
 [`crypto.createDiffieHellman()`][] function.
 
-```js
-const crypto = require('crypto');
-const assert = require('assert');
+```mjs
+import { strictEqual } from 'assert';
```

Comment:
As I said in https://github.com/nodejs/node/pull/37607, I have a preference for default import when using `assert`.
```suggestion
import assert from 'assert';
```